kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterExecutors.java

```diff
@@ -8,7 +8,15 @@
 
 import java.util.concurrent.ScheduledExecutorService;
 
+/**
+ * Executors available to the Filter
+ */
 public interface FilterExecutors {
```

Comment:
I wonder if this is really needed, or if the method could be moved to the `FilterConstructContext`?

Reply:
I imagined adding at least a blocking executor to this interface, so a large shared pool for if a contributor wanted to do any kind of slow/blocking background task. At the moment it's a bit thin with just the one type of Executor I know.

Reply:
maybe better to start on FilterConstructContext and can reintroduce it later if we do end up with a lot of executors being made available.

Reply:
Yeah, we need to think through what requires or justifies having a separate interface for it, v.s. a few methods on the context.